### DISCUSSION TITLE
Does the honkit search involve any external networking? Is it just parsing internal files and text?

### DISCUSSION THREAD

**Discussion by takeaki-m** [honkit/honkit#384] [Q&A] [answered]
We thank honkit for the simplicity and daily maintenance with honkit!

I'm considering use honkit in internal documentation. 
So I want to check if there is exchanging data with external external networking with using hokit.
the document contains sensitive information and we do not want to share any data with external.
I assume that honkit's default search function only parses the page and text, so no external communication occurs. 

Is the understanding correct?

**Comment by azu** [comment, marked as answer]
Yes.
Default search plugin(https://github.com/GitbookIO/plugin-search) use local index file via Lunr.
This index file is generated when honkit build the document.

For example, https://javascript.sumankunwar.com.np/ use https://javascript.sumankunwar.com.np/en/search_index.json as index file.

**Reply by takeaki-m** [reply]
Thank you for your reply. I'm introducing honkit to my project.
